playwright/README.md

```
@@ -0,0 +1,430 @@
+npm run check-types:playwright
+```
+
+### Single-Process Rule
+
+Never run two concurrent `npx playwright test` commands. Global setup creates shared auth files. Use `--workers=N` for parallelism within a single process.
+
+---
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)